pkg/status/templates.go

```diff
@@ -0,0 +1,797 @@
+// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
Should this file be checked in? I'm noticing it appear on my working copy as well. Perhaps this was suppose to be `.gitignored`?

Reply:
indeed I will add it to .gitignore